packages/SwingSet/docs/vat-upgrade.md

```diff
@@ -128,23 +135,26 @@ initializeBar(barData);
 
 ## From Outside: the AdminNode Upgrade API
 
-The upgrade must be triggered by something holding the vat's `adminNode` object. This is typically the same thing that created the vat in the first place, but a common pattern is to hand the `adminNode` to some sort of governance object. The ability to upgrade a vat is also the ability to control its behavior, and any users who expect some particular vat behavior (e.g. they auditing some contract code) must take into account the possibility of upgrade, which means they'll care about who exactly can cause an upgrade to occur.
+The upgrade must be triggered by something holding the vat's "adminNode" control facet. This is typically the same thing that created the vat in the first place, but a common pattern is to hand it to some sort of governance object. The ability to upgrade a vat is also the ability to control its behavior, and any users who expect some particular vat behavior (e.g. when auditing some contract code) must take into account the possibility of upgrade, which means they'll care about who exactly can cause an upgrade to occur.
 
-Upgrades use bundlecaps, just like the initial `createVat()` call. So the first step of an upgrade is to install the v2 source bundle (unless it is a "cross-grade" that re-uses the original bundle). This uses `controller.validateAndInstallBundle()` from the outside, and `D(devices.bundle).getBundleCap()` from the inside. See docs/bundles.md for details.
+Upgrades use bundlecaps, just like the initial `createVat()` call. So the first step of an upgrade is to install the v2 source bundle (unless it is a "null upgrade" that re-uses the original bundle). This uses `controller.validateAndInstallBundle()` from the outside, and `D(devices.bundle).getBundleCap()` from the inside. See [bundles.md](./bundles.md) for details.
```

Comment:
Let's refine that third sentence: "This uses .. from the outside, and *then* .. from the inside". The way I originally wrote it makes it sound like you can choose either one, but in fact you have to use both, in succession.